app/models/commits_service.rb

```diff
@@ -0,0 +1,8 @@
+class CommitsService
+
+  def self.find_commits(username, repository)
+    response = Faraday.get("https://api.github.com/repos/#{username}/#{repository}/commits?client_id=#{ENV['client_id']}&client_secret=#{ENV['client_secret']}")
```

Comment:
@Dpalazzari you should just need to pass the access_token with the request rather than the client_id and client_secret every time. that access_token is specific to your registered app and the user who logged in. check out the docs on this [here](https://developer.github.com/v3/#authentication)